app/services/waste_exemptions_engine/determine_area_service.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module WasteExemptionsEngine
+  class DetermineAreaService < BaseService
+    include CanCheckEastingAndNorthingValues
+
+    def run(easting:, northing:)
+      return unless valid_coordinates?(easting, northing)
+
+      lookup_area(easting, northing)
+    end
+
+    private
+
+    def lookup_area(easting, northing)
+      response = DefraRuby::Area::PublicFaceAreaService.run(easting, @orthing)
```

Comment:
```suggestion
      response = DefraRuby::Area::PublicFaceAreaService.run(easting, northing)
```
☝️ are we missing test coverage here?